src/main/scala/zio/zmx/server/ZMXClient.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package zio.zmx
+package zio.zmx.server
```

Comment:
Out of interest why this change in package?

Reply:
Just to separate implementation between the protocol server (`server`) and the one for `metrics`. We can change that if you prefer some other way.

Reply:
Good idea, got it now .... all good, happy with it. Ready for review?

Reply:
rebase